### PR TITLE
Adding known issue

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/26752

After reading the statement that you provided, it seemed more appropriate to add this as a known issue

### CODE REVIEW

File: governance/custom_template.adoc [hunk 1]
```diff
@@ -166,7 +166,7 @@ spec:
 [#lookup-func]
 === lookup function
 
-The `lookup` function returns the Kubernetes resource as a JSON compatible map. Note that if the requested resource does not exist, an empty map is returned.
+The `lookup` function returns the Kubernetes resource as a JSON compatible map. Note that if the requested resource does not exist, an empty map is returned. If the resource does not exist and the value is added to another template function, you might get the following error, `invalid value; expected string`.
```
Comment (mprahl) [inline governance/custom_template.adoc]:
```suggestion
The `lookup` function returns the Kubernetes resource as a JSON compatible map. Note that if the requested resource does not exist, an empty map is returned. If the resource does not exist and the value is provided to another template function, you might get the following error, `invalid value; expected string`.
```

File: governance/custom_template.adoc [hunk 2]
```diff
@@ -166,7 +166,9 @@ spec:
 [#lookup-func]
 === lookup function
 
-The `lookup` function returns the Kubernetes resource as a JSON compatible map. Note that if the requested resource does not exist, an empty map is returned. View the following syntax for the function:
+The `lookup` function returns the Kubernetes resource as a JSON compatible map. Note that if the requested resource does not exist, an empty map is returned. If the resource does not exist and the value is provided to another template function, you might get the following error, `invalid value; expected string`.
```
Comment (oafischer) [inline governance/custom_template.adoc]: I think we can leave out the "Note that" unless we want to break it out into a special tag.

Comment (oafischer) [inline governance/custom_template.adoc]: Should there be a colon instead of comma after error?

**Review by oafischer — APPROVED** [review]
Couple of comments to consider but I'll leave the decisions up to you, approving.

**Comment by dockerymick** [comment]
Thanks for your review and approval! I made updates based on your review. Merging this PR

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -926,6 +926,11 @@ While installation into an ARM environment is supported, operators that are depl
 * link:https://github.com/stolostron/policy-collection/blob/main/stable/SI-System-and-Information-Integrity/policy-imagemanifestvuln.yaml[{product-title-short} policy for the Quay Container Security Operator]
 * link:https://github.com/stolostron/policy-collection/blob/main/stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml[{product-title-short} policy for the Compliance Operator] 
 
+[#lookup-template-error]
```
Comment (mprahl) [inline release_notes/known_issues.adoc:929]: @dockerymick, I wouldn't consider this a known issue. This is just warning the user of valid behavior that they may not expect.

File: governance/custom_template.adoc (hunk 2 above)
Comment (dockerymick) [reply governance/custom_template.adoc]: I think either option is fine, however I agree that a colon is better

File: release_notes/known_issues.adoc (hunk 3 above)
Comment (dockerymick) [reply release_notes/known_issues.adoc:929]: Spoke through Slack and received clarification. Agreed to remove this as a known issue

**Pull request merged**